### DISCUSSION TITLE
h-screen not work

### DISCUSSION THREAD

**Discussion by zhugexinxin** [tailwindlabs/tailwindcss#11141] [Help]
node v14.21.3
"tailwindcss": "^3.3.2"

<img width="466" alt="image" src="https://user-images.githubusercontent.com/42615243/235675646-4f3555ba-b22a-48f9-b321-9e1dd6ad9894.png">
<img width="560" alt="image" src="https://user-images.githubusercontent.com/42615243/235675851-783aa634-7c15-4b9d-9e7e-d0338b603500.png">

**Comment by zhugexinxin** [comment]
w-96 is the same

**Reply by scheinercc** [reply]
Have you tried a hard refresh? Sometimes the css file is outdated, e.g. when using hot reloading.

**Comment by adamwathan** [comment]
Can you create a reproduction? My guess is that this file isn't properly covered by your [content configuration](https://tailwindcss.com/docs/content-configuration).
